README.md

```diff
@@ -118,8 +140,13 @@ The main purpose here is for backwards compatibility either for renaming categor
 for changing command access levels by moving it from one category to another.  It's recommended that
 such aliases be used only temporarily for version transitions.
 
+[//]: # If we want to discourage this, maybe we don't support it at all. I don't think it would be
+[//]: # hard to work around this (e.g. just assign the same code handler to multiple command strings)
```

Comment:
The reason to support it is that we often put out a release where we need compatibility for the pre-fork version.  So, for example, if `foo.bar` gets moved in loki v8.0.0 to `foobar.foo` then the initial release needs to still support it; once the hardfork is over then (e.g. in the first post-fork release, say v8.1.0) we can remove the alias.